src/Symfony/Bundle/FrameworkBundle/Resources/views/Form/attributes.html.php

```diff
@@ -1 +1,9 @@
-<?php echo $view['form']->block($form, 'widget_attributes') ?>
+<?php foreach ($attr as $k => $v): ?>
+<?php if (in_array($k, array('placeholder', 'title'), true)): ?>
```

Comment:
I think it's worth doing a micro optim here and turn this into:
`if ('placeholder' === $k || 'title' === $k):`